flutter/rsvp/lib/Models/event.dart

```diff
@@ -7,6 +7,7 @@ class Event {
   String _location;
   int _minNum;
   bool _status;
+  bool _isCreatedEvent;
```

Comment:
This shouldn't be in `Event`. It should be a property in `EventViewModel` which is passed in to its constructor when it is created in `MyEventsViewModel`. `MyEventsViewModel` should be responsible for creating `EventViewModel`s from the list of created and joined events and merging them into a single list (right now this happens in `EventRepository`, instead `EventRepository` should have two methods, one for getting created events and one for joined events).
This doesn't make sense to be in the `Event` class because it is not a property of the event, but our logic for displaying a single list for two kinds of events.